OsuRTDataProvider.csproj

```diff
@@ -71,6 +71,9 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Sync, Version=2.18.5.0, Culture=neutral, PublicKeyToken=null">
+      <HintPath>F:\syna app\Sync.exe</HintPath>
```

Comment:
Remove this reference because it contains your private info and specify version of .dll file reference.